lib/components/Header.js

```diff
@@ -1,56 +1,55 @@
 import React, { Component } from 'react';
 import { Link } from 'react-router';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
 import firebase from 'firebase';
 
 import SignIn from './SignIn';
 import * as actions from '../actions/auth';
 
-const Header = (props) => {
 
-   const { auth } = props;
+class Header extends Component {
 
-  return(
-    <header className="MainHeader">
-      <h1 className="MainTitle">
+  render() {
+    // const { status } = this.props;
+
+    console.log(this.props.status);
+    return(
+      <header className="MainHeader">
+        <h1 className="MainTitle">
+            <Link
+              to="/"
+              className="MainTitle"
+            >
+              CO Kid Hikes
+            </Link>
+          </h1>
+
+        <section className="HeaderLinks">
+        { status = 'LOGGED_IN' ?
```

Comment:
Yea I wasn't sure if that would solve all your problems or not, I was hoping it would!! I have to finish up some curriculum work tonight but can take a look at this further in the morning for ya.